exercises/series/description.md

```diff
@@ -13,7 +13,7 @@ And the following 4-digit series:
 - "4914"
 - "9142"
 
-And if you ask for a 6-digit series from a 5-digit string, you deserve whatever you get.
+Trying to extract a 6-digit series from a 5-digit string should result in an error.
```

Comment:
Is "should result in an error" something that should be track specific behavior?